extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/AnnotationsTransformerBuildItem.java

```diff
@@ -2,7 +2,20 @@
 
 import io.quarkus.arc.processor.AnnotationsTransformer;
 import io.quarkus.builder.item.MultiBuildItem;
+import org.jboss.jandex.AnnotationTarget;
 
+/**
+ * Allows to add, remove or alter annotations on CDI components.
```

Comment:
> not gramatically correct

You got me there, I'll fix that.

> could be used for non-CDI classes as well.

I thought about writing it more generally...but we cannot be sure that if you use it elsewhere, it will be taken into account, right? Only extensions that operate with `AnnotationStore` will see the changes :-/

Reply:
> Only extensions that operate with AnnotationStore will see the changes :-/

Yes, and ideally all extensions that read annotations should use the `AnnotationStore`. I know it's a utopia and the API is not ready for this kind of usage but still I'd like to keep it as general as possible ;-).

Reply:
Fair enough, I'll edit that.